confidential/confidential_test.go

```diff
@@ -79,3 +105,92 @@ func TestUnblindOutput(t *testing.T) {
 		assert.Equal(t, hex.EncodeToString(output.ValueBlindingFactor[:]), valueBlindingFactor)
 	}
 }
+
+func TestFinalValueBlindingFactor(t *testing.T) {
+	err := setUp()
+	if !assert.NoError(t, err) {
+		t.FailNow()
+	}
+
+	vectors := tests["FinalValueBlindingFactorTC"].([]interface{})
+	for _, testVector := range vectors {
+		v := testVector.(map[string]interface{})
+
+		inValuesSlice := v["inValues"].([]interface{})
+		inValues := make([]uint64, 0)
+		for _, v := range inValuesSlice {
```

Comment:
I would change the name of the variable since `v` is already declared before. `vv` is just fine to me.

Reply:
I changed it to val.